docs/Walkthrough/Working with the Catalog.md

```diff
@@ -28,15 +28,15 @@ Catalogs also enable you to search the contents of your bucket. We support both
 
 ### Upload package
 
-You can upload a new package providing the name of the package, commit message, files, metadata, and workflow.
+You can upload a new package providing the name of the package, commit message, files, metadata, and [workflow](../Advanced Features/workflows.md).
```

Comment:
Weird. This link wasn't transformed too https://github.com/quiltdata/quilt/blob/fcb3172dadfa63be85322e31d567b9d6dbfee05e/docs/Walkthrough/Working%20with%20the%20Catalog.md

Reply:
Maybe it's because of the commit hash in the URL.